config/config.go

```diff
@@ -679,6 +680,7 @@ type EC2SDConfig struct {
 	SecretKey       string   `yaml:"secret_key,omitempty"`
 	RefreshInterval Duration `yaml:"refresh_interval,omitempty"`
 	Port            int      `yaml:"port"`
+	SubnetSeparator string   `yaml:"subnet_separator,omitempty"`
```

Comment:
For the sake of consistency with the default for consul, I think we should stick with semicolon.

We're going to have quite a few of these (e.g. security groups will need similar handling, though those allow semicolons) so consistency will make things simpler for users.